polytope_server/common/authentication/mongodb_authentication.py

```diff
@@ -37,9 +36,12 @@ def __init__(self, name, realm, config):
         host = config.get("host", "localhost")
         port = config.get("port", "27017")
         collection = config.get("collection", "users")
+        username = config.get("username")
+        password = config.get("password")
+        tls = config.get("tls", False) == True
```

Comment:
`class MongoAuthentication `now appears in two places, is that the aim?

Reply:
actually name is slightly different, but still not sure why both are needed?

Reply:
Usually, authentication checks that the user is who he/she claims to be. Authorization checks that the user actually has access to the resource that the user is trying to access. For consistency, DB should be in both names. However, I think Stephan's goal is to make as few changes as possible.

Reply:
Every component (request store, auth, ...) has its own mongo client. For now the the goal was to just add a way of providing credentials to the mongo client.

Reply:
Ok thanks for the explanation